openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/objectRepository/Webhooks.java

```diff
@@ -0,0 +1,68 @@
+package org.openmetadata.catalog.selenium.objectRepository;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Webhooks {
+  WebDriver webDriver;
+
+  public Webhooks(WebDriver webDriver) {
```

Comment:
You can use lombok RequiredArgsConstructor

Reply:
Updated lombok @RequiredArgsConstructor